clang/lib/Driver/ToolChains/RTEMS.cpp

```diff
@@ -225,6 +223,11 @@ void RTEMS::AddClangSystemIncludeArgs(const ArgList &DriverArgs,
     SmallString<128> P(D.SysRoot);
     llvm::sys::path::append(P, "include");
     addExternCSystemInclude(DriverArgs, CC1Args, P.str());
+  } else {
+    // Add a default RTEMS target directory to the include path
+    SmallString<128> P(D.InstalledDir+"/../"+getTripleString());
```

Comment:
I would run `git-clang-format -f HEAD` to get consistent formatting.

Reply:
Done